File: frontend/src/constants/branding.js
// Fork branding. The product name is deliberately one constant so that a rename
// touches this file only. Technical identifiers (agent binary, service names,
// config paths, env prefix, webhook header) are NOT derived from it.
export const PRODUCT_NAME = "amanIT PatMan";
export const PRODUCT_NAME_SHORT = "PatMan";
export const VENDOR_NAME = "amanIT GmbH";
export const VENDOR_URL = "https://amanit.swiss";

// Public source repository of this fork (AGPL v3 section 13 offer).
export const SOURCE_REPO_URL = "https://github.com/lamanese/PatchMon";
export const SOURCE_CODE_URL = `${SOURCE_REPO_URL}/tags`;
export const DOCS_BASE_URL = `${SOURCE_REPO_URL}/blob/feat/remote-reboot/docs`;

// Tag that holds the source of a given running version (tags are v<version>).
export const sourceUrlForVersion = (version) =>
	version ? `${SOURCE_REPO_URL}/tree/v${version}` : SOURCE_CODE_URL;

File: server-source-code/internal/branding/branding.go
// Package branding holds the user-facing product name of the fork.
//
// The name is deliberately one set of constants so that a rename touches this
// file only. Technical identifiers (agent binary name, service names, config
// paths, the PM_ environment prefix, the X-PatchMon-Signature webhook header,
// the "PatchMon Agent v" banner parsed by the server) are NOT derived from it
// and must not be changed here.
package branding

const (
	// ProductName is the full product name shown in titles, mails and PDFs.
	ProductName = "amanIT PatMan"
	// ProductNameShort is the short form used where space is tight.
	ProductNameShort = "PatMan"
	// VendorName is the operator named in footers and as PDF author.
	VendorName = "amanIT GmbH"
	// VendorURL is linked from footers.
	VendorURL = "https://amanit.swiss"
)
